template.tpl

```diff
@@ -0,0 +1,550 @@
+﻿___TERMS_OF_SERVICE___
+
+By creating or modifying this file you agree to Google Tag Manager's Community
+Template Gallery Developer Terms of Service available at
+https://developers.google.com/tag-manager/gallery-tos (or such other URL as
+Google may provide), as modified from time to time.
+
+
+___INFO___
+
+{
+  "type": "TAG",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "Matomo Tracking Tag for Matomo Cloud",
+  "categories": ["ANALYTICS"],
+  "brand": {
+    "id": "brand_dummy",
```

Comment:
I'm not familiar with GTM templates. Is this a placeholder that needs to be updated or was this left intentionally?

Reply:
@snake14 Wasn't left intentionally, since the UI never gave any option to update this, so we never updated it.
I will try by changing this field and see if it works

Reply:
@snake I changed the id to `brand_matomo` and seems to work as expected